app/views/steps/check/results/shared/_disclaimer.en.html.erb

```diff
@@ -25,5 +25,5 @@
   start a new check.
 </p>
 
-<%= link_button :restart_check, root_path, class: 'govuk-button ga-pageLink govuk-!-margin-top-2',
```

Comment:
There are many other `root_path` in use in the service, I think a better solution is to redefine `root` in the `routes.rb` to point to the new controller#action

That way there is no need to change one by one in all the views, etc.